### ISSUE TITLE
GNOME 3.22

### ISSUE THREAD

**Issue by sternenseemann** [NixOS/nixpkgs#19929]
[GNOME 3.22](https://www.gnome.org/news/2016/09/gnome-3-22-released-the-future-is-now/) has been released (GNOME 3.22.1 to be precise).

Is it intended to update the NixOS version of GNOME?

I already have given it a try to package GNOME 3.22 (mainly because I am interested in the HTTPSeverywhere support of epiphany) using `./maintainers/scripts/gnome.sh` but I ran into an annoying build problem which even might be an upstream issue.

**TL;DR:** GNOME 3.22 is out, is it wanted to update GNOME 3.20 to GNOME 3.22 on NixOS?

**Comment by NeQuissimus** [comment]
#19081 is what you want :)

**Comment by RonnyPfannschmidt** [comment]
https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+gnome+22+is%3Aclosed is pretty much all the stuff you need to know

please use the search feature - they actually started to remove gnome 3.22 as well

**Comment by sternenseemann** [comment]
Oh, I see. I briefly searched for gnome-related issues but only for issues unfortunately.
